source/mir/random/flex/internal/area.d

```diff
@@ -186,13 +190,13 @@ body
         }
         else if (iv.c == 1)
         {
-            area = S(0.5) * sh.a * intLength * (z + 2);
+            area = S(0.5) * sh.a * ivLength * (z + 2);
```

Comment:
> But \tilde{f}(x) is not the same as T_c(x) ... ?

As far as I understand, is `T_c(x)` a family of two transformations
- `log(x)` for `c == 0`
- `sgn(c) * x^c` for `c != 0`

and `\tilde{f}{x} == T_c(f(x))`, so `f(x)` with the applied transformation.

It's a very confusing in the paper because they define a lot of special cases to avoid numerical errors in the common case.